fastn-core/src/http.rs

```diff
@@ -735,7 +735,6 @@ pub fn user_err(
     }
 
     let resp = serde_json::json!({
-        "data": null,
         "errors": json_error,
     });
```

Comment:
We should return a key `success: bool` so based on key we can easily detect if it was success or failure, else we have to do existence checks.